packages/flutter/lib/src/widgets/scrollbar.dart

```diff
@@ -622,6 +622,11 @@ class ScrollbarPainter extends ChangeNotifier implements CustomPainter {
       return false;
     }
 
+    // The thumb is not able to be hit when invisible.
```

Comment:
It doesn't look like we're really checking if the thumb is visible. We're returning false if it's not possible to scroll (here and elsewhere).

Maybe define
```dart
bool get _lastMetricsAreScrollable => _lastMetrics!.minScrollExtent != _lastMetrics!.maxScrollExtent;
```